src/test/java/ancaMarian/tests/tema5/DemoQA.java

```diff
@@ -0,0 +1,207 @@
+package ancaMarian.tests.tema5;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class DemoQA {
+    private static String url;
+    private static ChromeDriver driver = null;
+
+    DemoQA(String url, ChromeDriver driver) {
+        this.url = url;
+        this.driver = driver;
+        driver.get(url);
+    }
+    public static void checkTextBox() {
+        WebElement textBox = driver.findElement(By.xpath("//*[@id='item-0']/span"));
+        textBox.click();
+    }
+
+    public static void enterFullName() {
+        WebElement fullName = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/form/div[1]/div[2]/input"));
```

Comment:
Maybe I forgot to mention but it would be desirable to change all absolute xpaths with relative ones. Thanks!

Reply:
Nu le pot schimba pentru ca am probleme cu pc-ul.